### ISSUE TITLE
Fix DNT build

### ISSUE THREAD

**Issue by tjjfvi** [paritytech/capi#358]
Currently failing with
```
error: Uncaught (in promise) 'Module not found "file:///.../capi/effect/mod.ts".'
```
But there may be other issues (such as `deps/smoldot_phantom.ts`)

**Comment by harrysolovay** [comment]
Ah yes, we haven't updated DNT since merging the new RPC impl. Will tackle this soon.

**Comment by harrysolovay** [comment]
Should be resolved as of #438
